atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -134,6 +134,15 @@ const CrashesByMode = () => {
       }, 0);
     });
 
+  const adjustModeData = (modeData) => {
+    // TEMPORARY MEASURE!!
+    // Manually adjust mode data displayed for 2022 pedestrian fatalities to match VZ Team's analysis
+    if (crashType.name === "fatalitiesAndSeriousInjuries" || crashType.name === "fatalities") {
+      modeData[1].data[3] = modeData[1].data[3] + 1
```

Comment:
The path of least resistance is what makes the river crooked.

I want to reiterate my support for this measure in as much as we follow up on finding the real, root cause. Thank you Tilly for this work, and to everyone for our discussion where we continue to value generalized solutions that fully comprehend and address discrepancies in calculations.

Reply:
Another thought - I think this will offset "last year" and not 2022 explicitly. In this particular case, I think that is fine, as this is temporary, but we'll be scratching our heads pretty good if we don't address this before 2024.

Reply:
Thanks, @frankhereford !! re: your comment about "last year" - that was my main reservation with this code, but since the data is calculated as current year minus a year, minus two years, etc., this was the only way i could think of to do this. I could implement a failsafe that disables this when current year is 2024, but i thought the chances of us getting there before we implement a long-term fix were pretty slim. still, happy to do so if that makes everyone more comfortable.

Reply:
You're way ahead of me! I think this is totally fine to have a ticking "reminder" that we need to address it. 

Makes me wish we could put a "become bright read and start flashing at a certain date" reminder on issues.. GitHub, are you out there? Can you hear me? 

Thanks Tilly!!